BoardConfig.mk

```diff
@@ -57,12 +57,16 @@ BOARD_FLASH_BLOCK_SIZE := 512
 BOARD_VENDORIMAGE_FILE_SYSTEM_TYPE := ext4
 BOARD_SYSTEMIMAGE_FILE_SYSTEM_TYPE := ext4
 BOARD_BOOTIMAGE_PARTITION_SIZE := 30720000
+BOARD_VENDOR_BOOTIMAGE_PARTITION_SIZE := 30720000
 BOARD_USERDATAIMAGE_PARTITION_SIZE := 3145728000  # 3145728000 = 3000 MiB
 
+BOARD_INCLUDE_DTB_IN_BOOTIMG := false
+PRODUCT_COPY_FILES += device/google/cuttlefish/dtb.img:dtb.img \
```

Comment:
Hi @arminn, you've explained to me the motivation for the addition of this line. Could you, please, add the same explanation to the commit message? The presence of this line is not self-explaining, thus, requires additional "decoding". ))

Reply:
Done